ncm-metaconfig/src/main/metaconfig/logstash/1.2/tests/regexps/forwarder/base

```diff
@@ -6,8 +6,8 @@ multiline
 ^\{$
 ^\s{4}"network": \{$
 ^\s{8}"servers": \[$
-^\s{12}"myhost:12345",$
-^\s{12}"myhost2:12346"$
+^\s{12}"myhost.example.com:12345",$
```

Comment:
Both are valid, see https://tools.ietf.org/html/rfc2606#page-2, but it might pay to be consistent.